cmd/fynedesk_runner/log.go

```diff
@@ -1,34 +1,49 @@
 package main
 
 import (
-	"io"
+	"fmt"
 	"os"
 	"path/filepath"
+	"time"
 
 	"fyne.io/fyne"
 )
 
-func logPath() string {
-	return logPathRelativeTo(systemLogDir())
+func crashLogPath() string {
+	return crashLogPathRelativeTo(systemLogDir())
 }
 
-func logPathRelativeTo(parent string) string {
+func crashLogPathRelativeTo(parent string) string {
+	path := filepath.Join(logDir(parent), "fynedesk")
+	now := time.Now().Format(time.RFC3339)
+	return fmt.Sprintf("%s-crash-%s.log", path, now)
+}
+
+func logDir(parent string) string {
 	cacheDir := filepath.Join(parent, "fyne", "io.fyne.fynedesk")
 	err := os.MkdirAll(cacheDir, 0700)
 	if err != nil {
 		fyne.LogError("Could not create log directory", err)
 	}
 
-	return filepath.Join(cacheDir, "fynedesk.log")
+	return cacheDir
+}
+
+func logPath() string {
+	return logPathRelativeTo(systemLogDir())
+}
+
+func logPathRelativeTo(parent string) string {
+	return filepath.Join(logDir(parent), "fynedesk.log")
 }
 
-// openLogWriter returns a stream for stdOut and stdErr of the process we run
-func openLogWriter() (io.WriteCloser, io.WriteCloser) {
+// openLogWriter returns the log file for stdOut and stdErr of the process we run
```

Comment:
This comment seems incorrect now that it only returns stderr.

Reply:
Hmm, maybe - it's not incorrect though, as both streams should be writing to it?

Reply:
Updated the comment, removing possible confusion